website/sidebars.json

```diff
@@ -1,20 +1,39 @@
 {
-  "docs": {
-    "Overview": [
-      "specification",
-      "implementations",
-      "server-discovery",
-      "faq"
-    ],
-    "Extensions": [
-      "extensions/jvm",
-      "extensions/scala",
-      "extensions/sbt",
-      "extensions/java",
-      "extensions/maven"
-    ],
-    "Bindings": [
-      "bindings/java"
-    ]
-  }
+  "docs": [
+    {
+      "type": "doc",
+      "label": "Specification",
+      "id": "specification"
+    },
+    {
+      "type": "category",
+      "label": "Concepts",
```

Comment:
Also I don't think concepts is clear here. I think we should stick to overview. Plus then we don't need to change the existing urls.

Reply:
unfortunately to my knowledge it is impossible to setup autogenerate without providing a specific folder

Reply:
Regarding redirects - I don't think we need to be so paranoid about them. Even if for some reason someone decided to save a link to the FAQ, finding the new one is rather straightforward. We don't have that many pages anyway.

Reply:
What I mean is can't this `label` just be `Overview`?

Reply:
And regarding the redirects, there is a built in way to add them. I'd recommend just adding the redirects especially seeing you just changed probably the most visited page on the site.